go/kbfs/simplefs/simplefs.go

```diff
@@ -587,42 +586,49 @@ func (k *SimpleFS) refreshSubscription(
 		return nil
 	}
 
+	tlfType, tlfName, _, _, err := remoteTlfAndPath(path)
+	if err != nil {
+		return err
+	}
+	tlfPathFromGUI := stdpath.Join("/keybase", tlfType.PathString(), tlfName)
```

Comment:
This is necessary now because links in chat means user can go to a path inside a TLF without going through its root. So the first path SimpleFS sees about a TLF isn't necessarily the TLF root.